tests/model_registry/image_validation/conftest.py

```diff
@@ -0,0 +1,20 @@
+from typing import Generator, Any
+
+import pytest
+from kubernetes.dynamic import DynamicClient
+from pytest_testconfig import config as py_config
+
+from ocp_resources.pod import Pod
+from tests.model_registry.constants import MR_INSTANCE_NAME
+from utilities.general import wait_for_pods_by_labels
+
+
+@pytest.fixture(scope="function")
+def model_registry_instance_pod(admin_client: DynamicClient) -> Generator[Pod, Any, Any]:
```

Comment:
maybe better a name that indicates that the labels are used to get the pod? `model_registry_instance_pod_by_label` or something similar